### PR TITLE
Add numRows to Textarea component

### PR DESCRIPTION
**Pull request by gloriakim-optimizely**
Currently there is no way to specify the height of the Textarea, which means all Textareas are fixed to a height of 62px. For the `textbox` used in the create/edit Recommender dialog (for filters) to be replaced with this OUI component, a different height (or `numRows`) needs to be supported.

Without `numRows` prop:
Screen Shot 2019-08-14 at 11 33 18 AM

With `numRows` prop set to 15:
Screen Shot 2019-08-14 at 11 23 06 AM

### CODE REVIEW

**Comment by gloriakim-optimizely** [comment]
Should this change be under `44.5.1 - 2019-08-14` inside the CHANGELOG?

**Comment by ayeshakmaz** [comment]
> Should this change be under `44.5.1 - 2019-08-14` inside the CHANGELOG?

@gloriakim-optimizely Nah, just under `#Unreleased`, and before `44.5.1 - 2019-08-14`

**Comment by gloriakim-optimizely** [comment]
Gotcha thanks!

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,7 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 This file is similar to the format suggested by [Keep a CHANGELOG](https://github.com/olivierlacan/keep-a-changelog).
 
 ## Unreleased
+- [Feature] Add `numRows` prop for **Textarea** component
```
Comment (ayeshakmaz) [inline CHANGELOG.md]: Last nit: can you add the link to the PR (see below for an example)

**Comment by gloriakim-optimizely** [comment]
Done! Thanks for reviewing!

**Pull request merged**